kai/service/incident_store/psql.py

```diff
@@ -279,29 +284,31 @@ def load_report(self, app: Application, report: Report) -> tuple[int, int, int]:
 
             self.conn.autocommit = False
             for si in solved_incidents:
-                file_path = os.path.join(
-                    repo_path,
-                    # NOTE: When retrieving uris from the report, some of them
-                    # had "/tmp/source-code/" as their root path. Unsure where
-                    # it originates from.
-                    unquote(urlparse(si[4]).path).removeprefix(
-                        "/tmp/source-code/"  # trunk-ignore(bandit/B108)
-                    ),
-                )
+                # NOTE: When retrieving uris from the report, some of them
+                # had "/tmp/source-code/" as their root path. Unsure where
+                # it originates from.
+                file_path = remove_known_prefixes(unquote(urlparse(si[4]).path))
+                # file_path = os.path.join(
+                #     repo_path,
+                #     in_repo_path,
+                # )
```

Comment:
As long as `git diff` works with the in-repo path as opposed to the full path then I'm good with this. Just pointing out the potential functionality difference.

Reply:
Yeah, I believe that the `Repo` object handles the relative path stuff